<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign</class>
 <widget class="QDialog" name="Sign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>241</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>user:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ageLabel">
        <property name="text">
         <string>password:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>repassword:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>email:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="userLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="repasswordLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="emailLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>178</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="finishButton">
      <property name="text">
       <string>完成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>178</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>完成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
